setup-homebrew/main.sh

```diff
@@ -136,7 +136,7 @@ fi
 ohai "Fetching Homebrew/brew..."
 if [[ "$GITHUB_REPOSITORY" =~ ^.+/brew$ ]]; then
     cd "$HOMEBREW_REPOSITORY"
-    git remote set-url origin "https://github.com/$GITHUB_REPOSITORY"
+    git remote set-url origin "https://${GITHUB_SERVER_URL}/$GITHUB_REPOSITORY"
```

Comment:
The documentation states `GITHUB_SERVER_URL` includes `https://`: https://docs.github.com/en/actions/reference/variables-reference

```suggestion
    git remote set-url origin "${GITHUB_SERVER_URL}/$GITHUB_REPOSITORY"
```